ci_build_images/msan.Dockerfile

```diff
@@ -0,0 +1,29 @@
+# this is to create images with MSAN for BB workers
+WORKDIR /tmp/msan
+
+COPY build-msan2.sh .
+
+RUN curl -sL https://apt.llvm.org/llvm-snapshot.gpg.key | gpg --dearmor -o /usr/share/keyrings/llvm-snapshot.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/llvm-snapshot.gpg] \
+    http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" > /etc/apt/sources.list.d/llvm-toolchain.list \
+    && echo "deb-src [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/llvm-snapshot.gpg] \
+    http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" >> /etc/apt/sources.list.d/llvm-toolchain.list \
+    && apt-get update \
+    && echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90forceyes \
```

Comment:
I don't think that this is needed since you use `apt-get -y` and we are in a non-interactive context.

Reply:
It was needed for `mk-build-deps -it`. I removed it and changed the commands to mark pass `yes` explicitly, like:
`mk-build-deps -it 'apt-get -o Debug::pkgProblemResolver=yes --no-install-recommends --yes'`

Reply:
Ok I see and thanks for this better solution since it does not touch the general OS configuration.